.github/portfolio.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+</head>
+<body>
+    <div class="section">
+    <h2>Here are some projects that I've worked on</h2>coid
+    <img class="quiz" src="images/quiz.png" alt="hairstyles-quiz">
+    <img class="anime" src="images/animeDB.jpeg" alt="my hero academia anime search">
+    <a href="https://github.com/Diamondd01/AnimeDatabase">Anime Database</a>
+    <a href="https://github.com/Diamondd01/Capstone-Project">Hairstyles quiz</a>
+</div>
+<article>
```

Comment:
I appreciate this use of semantic elements! see if there are other ways to increase accessibility by using even more semantic tags - https://www.w3schools.com/html/html5_semantic_elements.asp